ethereum/eth/src/main/java/org/hyperledger/besu/consensus/merge/NewForkchoiceMessageListener.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.hyperledger.besu.consensus.merge;
+
+import org.hyperledger.besu.datatypes.Hash;
+
+import java.util.Optional;
+
+public interface NewForkchoiceMessageListener {
```

Comment:
I hate having a lot of naming feedback, but wouldn't `ForkchoiceMessageListener` work here?

Reply:
yeah, i actually used that in a previous version.